amy/workshops/forms.py

```diff
@@ -1064,11 +1057,17 @@ class EventsMergeForm(forms.Form):
     start = forms.ChoiceField(choices=TWO, initial=DEFAULT, widget=forms.RadioSelect)
     end = forms.ChoiceField(choices=TWO, initial=DEFAULT, widget=forms.RadioSelect)
     host = forms.ChoiceField(choices=TWO, initial=DEFAULT, widget=forms.RadioSelect)
+    sponsor = forms.ChoiceField(choices=TWO, initial=DEFAULT, widget=forms.RadioSelect)
     administrator = forms.ChoiceField(
         choices=TWO,
         initial=DEFAULT,
         widget=forms.RadioSelect,
     )
+    public_status = forms.ChoiceField(
```

Comment:
I don't quite understand what caused this change. Was public status not originally a part of the form? Is that new functionality for events?

Reply:
Yes. It's a miss from a few weeks back when this field was introduced. Thankfully this form is not being used too often. Good catch.